### ISSUE TITLE
[RFC] 'about-us' for a slug - message in frontend

### ISSUE THREAD

**Issue by horus68** [bolt/bolt#5039]
This language line is taken from messages.en_GB.yml
https://github.com/bolt/bolt/blob/master/app/resources/translations/en_GB/messages.en_GB.yml
`
"If there was a Block with 'about-us' for a slug, it would've been shown here. But there isn't one, so that's why you're seeing this placeholder. In the Bolt backend, go to 'check database' and 'Add some sample Records', to create these automatically.": `

If an user do not install Bolt with prefilled contents this is displayed in frontend.
It should be one or both ways:
- not displayed when no page with that slug
- changed the actual language string content

 - Relevant Bolt Version: [ 2.2 | 3.0 | master ]

**Comment by horus68** [comment]
Proposal form "json_borne" user on IRC chanel
`If there was a Block with 'about-us' for a slug, it would've been shown here. But there isn't one, so that's why you're seeing this placeholder. In the Bolt backend, go to 'check database' and 'Add some sample Records', to create these automatically.`

**Comment by Puddingboy** [comment]
I wish to offer a suggestion. Feel free to make adjustments as you please:

```
A Block with 'about-us' as slug property does not exist. If you create one you will find it here.<br>
Go to 'configuration(config?)' > 'check database' and select 'Add sample Records' to autogenerate these entries.
```

I have my doubts on my own suggestion, but I didn't want to whine about something without offering my own input.

**Comment by SahAssar** [comment]
It's a bit weird that that text refers to the 'Add sample Records' page since that will only create lorem ipsum records, not any with the slug 'about-us'... I'd say we should just remove that part from it and instruct users to create a Block with the slug 'about-us'.

**Comment by bobdenotter** [comment]
> page since that will only create lorem ipsum records, not any with the slug 'about-us'

Actually, it does.. When the dummy generator finds no blocks, it creates the blocks with the default names. :-)

**Comment by SahAssar** [comment]
@bobdenotter Oh wow, I had no idea about that :D

**Comment by bobdenotter** [comment]
Fixed in #5116
